src/browser/components/SettingsPage.jsx

```diff
@@ -470,6 +472,13 @@ const SettingsPage = createReactClass({
               xs={8}
             >
               <h2 style={settingsPage.sectionHeading}>{'Server Management'}</h2>
+              <div className='IndicatorContainer'>
+                <AutoSaveIndicator
+                  show={this.state.savingType === 'server'}
+                  savingState={this.state.savingState}
```

Comment:
To take @jasonblais suggestion, you would need to manage different `state` parameters for each indicator.